doc/build/faq/connections.rst

```diff
@@ -257,7 +257,7 @@ statement executions::
                 try:
                     fn(cursor_obj, statement, context=context, *arg)
                 except engine.dialect.dbapi.Error as raw_dbapi_err:
-                    connection = context.root_connection
+                    connection = context.root_connection.connection.dbapi_connection
```

Comment:
we can't assign connection to be the dbapi one, since later it's used as a sqlalchemy one.

let's just change the call to is_disconnect to use the dbapi one